docs/Agent/installation/ansible_based_installation.md

```diff
@@ -26,9 +22,9 @@ The same pbench-agent version must be installed on all the test systems that par
  export ANSIBLE_ROLES_PATH=$HOME/.ansible/collections/ansible_collections/pbench/agent/roles:$ANSIBLE_ROLES_PATH
  ```
 
-4. Create an inventory file (`~/.config/Inventory/myhosts.inv`) naming the hosts on which you wish to install pbench-agent and including information about where the config file and ssh key file can be found. Example inventory [file](assets/myhosts.inv).
+4. Create an inventory file (`~/.config/Inventory/myhosts.inv`) naming the hosts on which you wish to install pbench agent and including information about where the config file. Example inventory [file](assets/myhosts.inv).
 
-5. Create a playbook [file](assets/pbench_agent_install.yml)
+5. Create a playbook [pbench_agent_install.yml](https://github.com/distributed-system-analysis/pbench/blob/main/agent/ansible/playbooks/pbench_agent_install.yml) file.
```

Comment:
Maybe, "use the example playbook or reference it to customize your own"???

Reply:
fixed